dnp3/examples/outstation-tokio-main.rs

```diff
@@ -69,7 +70,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let mut value = 0.0;
 
     loop {
-        tokio::time::delay_for(Duration::from_secs(5)).await;
+        tokio::time::delay_for(Duration::from_secs(5000)).await;
```

Comment:
?

Reply:
I'm guessing this was to test the link status

Reply:
Oups, forgot to revert it back. Just force-pushed an update.